drivers/ethernet/Kconfig

```diff
@@ -3,24 +3,19 @@
 # Copyright (c) 2015 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
 
-menu "Ethernet Drivers"
+menuconfig ETH_DRIVER
+	bool "Ethernet Drivers"
+	default y
 	depends on NET_L2_ETHERNET
```

Comment:
A helper section explaining why this symbol is required would be needed, IMHO.
For instance, this would avoid that people adding new drivers and taking this one as example also add their own `FOO_DRIVER` symbol with no reason.

Reply:
BTW, do we actually need to show this selection to the user. Typically this symbol is disabled only by the network tests and not directly by the user, so the `Ethernet Drivers` text could be omitted.

Reply:
@erwango most drivers/<FOO>/Kconfig use a `menuconfig`.

Reply:
Ha, right :-)